pkg/apis/monitoring/v1beta1/doc.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 The prometheus-operator Authors
```

Comment:
This is a new file yet it says 2017, I assume its generated so I guess we should fix some generation there, do you mind opening an issue if this is the case, thanks!

Reply:
This file was actually manually created, I just copied it from the v1 folder I think, so I'll update it manually.

For generated files, they will have 2018 as the copyright year, so I've created #3546 for those just now.